modules/test/playwright/tests/change-tracking-web/publicationsUserCollaboration.spec.ts

```diff
@@ -0,0 +1,170 @@
+		user.id
+	);
+
+	await changeTrackingPage.goToReviewChanges(ctCollection.name);
+
+	await page.getByLabel('View Collaborators').click();
+
+	await page.getByLabel('Viewers can view. Viewers').click();
+
+	await page.getByRole('menuitem', {name: 'Admin'}).click();
+
+	const enterNameOrEmailTextbox = page.getByPlaceholder(
+		'Enter name or email address.'
+	);
+
+	await enterNameOrEmailTextbox.click();
+
+	await enterNameOrEmailTextbox.fill(user.emailAddress);
+
+	await page.waitForTimeout(300);
+
+	await page.getByRole('option', {name: user.name}).click();
+
+	await page.getByLabel('Submit').click();
+
+	await page.reload();
+
+	await journalEditArticlePage.goto();
+
+	const title = getRandomString();
+
+	await journalEditArticlePage.fillTitle(title);
+
+	await page.getByRole('button', {name: 'Publish'}).click();
+
+	await waitForSuccessAlert(
+		page,
+		`Success:${title} was created successfully.`
+	);
+
+	await performLogout(page);
+
+	await performLogin(page, user.alternateName);
+
+	await page.getByTestId('userPersonalMenu').click();
+
+	await page.getByRole('menuitem', {name: 'Notifications'}).click();
+
+	await expect(
+		page.getByText(
+			`has invited you to work on ${ctCollection.name} as a Admin.`
+		)
+	).toBeVisible();
+
+	await changeTrackingPage.goto();
+
+	await page.waitForLoadState('load');
+
+	await page.getByRole('button', {name: 'Actions'}).click();
+
+	await page.getByRole('menuitem', {name: 'Edit'}).click();
+
+	await page.getByPlaceholder('Enter the name of the').fill(title);
+
+	await page.getByRole('button', {name: 'Save'}).click();
+
+	await waitForSuccessAlert(page, 'Success:Successfully updated');
+
+	await page.getByRole('link', {name: title}).click();
+
+	await page.getByRole('link', {name: 'Publish'}).click();
+
+	await page.getByRole('button', {name: 'Publish'}).click();
+
+	await expect(page.getByText('History')).toBeVisible();
+
+	await expect(
+		page.locator('div').filter({hasText: title}).first()
+	).toBeVisible();
+
+	await waitForSuccessAlert(
+		page,
+		'Success:Your request completed successfully.'
+	);
+
+	await performLogout(page);
+
+	await performLogin(page, 'test');
+
+	await journalPage.goto();
+
+	await page.getByLabel(`Actions for ${title}`).click();
+
+	await page.getByRole('menuitem', {name: 'Delete'}).click();
+
+	await page.reload();
+
+	await productMenuPage.openProductMenuIfClosed();
+
+	await page.getByRole('menuitem', {name: 'Recycle Bin'}).click();
+
+	await page.getByLabel('Recycle Bin').getByTestId('app').click();
+
+	await expect(
+		page
+			.getByTestId('header')
+			.locator('div')
+			.filter({hasText: 'Recycle Bin'})
+			.nth(1)
+	).toBeVisible();
+
+	await page.getByLabel('Select All Items on the Page').check();
+
+	await page.getByRole('button', {name: 'Delete'}).click();
+
+	await page
+		.getByLabel('Delete- Loading')
+		.getByRole('button', {name: 'Delete'})
+		.click();
+
+	await waitForSuccessAlert(
+		page,
+		'Success:Your request completed successfully.'
+	);
```

Comment:
Add `com.liferay.journal.service.impl.JournalArticleServiceImpl#deleteArticle` to `JSONWebServicesJournalApiHelper` to delete the web content.

Reply:
Now the web content is being added within a site, that gets deleted afterwards, so there's no need to add this method anymore